src/integrations/thumbnail.ts

```diff
@@ -126,28 +124,22 @@ export const get: APIRoute = async ({ url }) => {
 		}	
 	} catch (error) {
 		// If everything breaks, use a default image
```

Comment:
tbh... I do wonder if we should instead take the approach of "if anything breaks, break"